packages/core/src/setup.ts

```diff
@@ -226,9 +270,61 @@ export function setup<
     TMeta,
     TConfig
   >;
+
+  assign: typeof assign<
+    TContext,
+    TEvent,
+    undefined,
+    TEvent,
+    ToProvidedActor<TChildrenMap, TActors>
+  >;
+  sendTo: <TTargetActor extends AnyActorRef>(
+    ...args: Parameters<
+      typeof sendTo<
+        TContext,
+        TEvent,
+        undefined,
+        TTargetActor,
+        TEvent,
+        TDelay,
+        TDelay
+      >
+    >
+  ) => ReturnType<
+    typeof sendTo<
+      TContext,
+      TEvent,
+      undefined,
+      TTargetActor,
+      TEvent,
+      TDelay,
+      TDelay
+    >
+  >;
+  sendParent: typeof sendParent<
+    TContext,
+    TEvent,
+    undefined,
+    AnyEventObject,
+    TEvent,
+    TDelay,
+    TDelay
+  >;
+  raise: typeof raise<TContext, TEvent, TEvent, undefined, TDelay, TDelay>;
+  log: typeof log<TContext, TEvent, undefined, TEvent>;
+  cancel: typeof cancel<TContext, TEvent, undefined, TEvent>;
+  stopChild: typeof stopChild<TContext, TEvent, undefined, TEvent>;
 } {
   return {
```

Comment:
this misses: `enqueueActions`, `emit` and `spawnChild`. We can ship without them but Im pointing this out as I'm not sure if it's a deliberate choice or accidental omission